docusaurus/docs-internals/editor-controls/data-controls/number.md

```diff
@@ -0,0 +1,53 @@
+# Number
+
+The number control in Brizy provides a straightforward number input field, allowing you to set minimum and maximum value limits and define the step size for value adjustments.
+This control should be used as `type: "number"`
```

Comment:
i think this part we can remove because below in the table already is described

![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/13def470-b959-489e-af8d-a0d93ecc8cfa)